test/setup.ts

```diff
@@ -151,7 +151,7 @@ declare global {
     type MockInstance<T, Y extends any[]> = JestMockInstance<(...args: Y) => T>;
 
     interface Mock<T = any, Y extends any[] = any>
-      extends Function, // eslint-disable-line @typescript-eslint/ban-types
+      extends Function,
```

Comment:
Unrelated change

Reply:
In my (docker/vs code) dev env `yarn test` stumbles over this because of
 `154:25  error  Unused eslint-disable directive (no problems were reported from '@typescript-eslint/ban-types')`

Am I doing something wrong, or should the eslint directive be reinstated?

Reply:
Maybe you needed to run `yarn` again to get the latest dependencies